src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/service/DocumentReceiver.java

```diff
@@ -62,7 +62,7 @@ public DocumentWithMetadata receive(
                     document,
                     documentWithDescription.getDescription().orElse(""),
                     dateUploaded,
-                    tag
+                    tag,"test"
```

Comment:
@burale please use empty String or null instead of `test`

Reply:
Resolved